libekb.C

```diff
@@ -115,6 +115,15 @@ static void get_HWPErrorInfo(const fapi2::ReturnCode& rc, HWP_ErrorInfo& hwp_err
     {
         HWCallout hwcallout_data;
         hwcallout_data.hwid = fapi2::plat_HwCalloutEnum_tostring(hwcallout->iv_hw);
+
+        // Workaround for clock failure.
+        // only PROC_REF_CLOCK failure is expected in BMC context.
+        if(hwcallout->iv_hw == fapi2::HwCallouts::HwCallout::PROC_REF_CLOCK)
```

Comment:
Need to remove unused  enums from https://github.com/open-power/libekb_p10/blob/main/hwpf/fapi2/src/plat/plat_utils.C#L146